ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/ThreeDSecureRenderType.kt

```diff
@@ -0,0 +1,9 @@
+package com.braintreepayments.api.threedsecure
+
+enum class ThreeDSecureRenderType {
```

Comment:
Were there doc strings for this?

Reply:
There was not :( It looks like these values map to types in the Cardinal SDK. I can do some digging and see if I can come up with some docs for the enum.

Reply:
If they didn't exist before that can be a later TODO 👍

Reply:
Sounds good. After a quick search, I wasn't able to find Cardinal documentation on these :(